install/kubernetes/helm/istio/templates/namespace.yaml

```diff
@@ -1,4 +0,0 @@
-apiVersion: v1
-kind: Namespace
-metadata:
- name: {{ .Release.Namespace }}
```

Comment:
Not sure what to do about this problem...

Reply:
@yusuoh can you point me to the script that tries to run this without helm? helm does not like this particular template.

Reply:
@mandarjog @kyessenov 
It is in #4612. I moved the namespace template out of Helm charts and copy the content to the head of every generated yamls in the Makefile.

Reply:
It's still there after your PR: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/templates/namespace.yaml

Reply:
Oh, the pr is not merged yet. I am waiting for the approval.

Reply:
Ok, whichever one gets first is not important then. This one is ready, too.